offload/liboffload/src/OffloadImpl.cpp

```diff
@@ -261,12 +297,21 @@ Error olGetDeviceInfoImplDetail(ol_device_handle_t Device,
     return Device == HostDevice() ? ReturnValue(OL_DEVICE_TYPE_HOST)
                                   : ReturnValue(OL_DEVICE_TYPE_GPU);
   case OL_DEVICE_INFO_NAME:
-    return ReturnValue(GetInfo({"Device Name"}).c_str());
+    if (Device == HostDevice())
+      return ReturnValue("Host");
```

Comment:
I wonder if we should do something to avoid having to special-case the host device everywhere in this function. Maybe a separate function that we delegate to when `Device == HostDevice()`. It could return an `UNSUPPORTED_ENUM` error for most properties since I don't think we really want to implement all of them.